crm_lead_currency/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "maintainers": [
         "luisg123v",
     ],
-    "version": "14.0.1.0.1",
+    "version": "14.0.1.0.0",
```

Comment:
This is not correct. You're setting a wrong version on the previous commit, to be then fixed in the next one.

As I said, that should be done previous to the pre-commit commit, as if they were introduced in 12.0.

Besides, title needs to be fixed:

`[IMP] crm_lead_views: update version and view identation`
- Module should be `crm_lead_currency`
- identation -> indentation

Reply:
@luisg123v I have set in the commit [[MIG] crm_lead_currency: Migration to 14.0] te version to 14.0.1.0.0, 
Also I have fixed the name of the last commit